<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertCourse</class>
 <widget class="QWidget" name="insertCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="classCapacityLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>课程容量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>课程时长（分钟）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>授课教练</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classDurationLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>难度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trainerIdLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classNameLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classDescriptionTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>课程描述</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classLevelComboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>初级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高级</string>
    </property>
   </item>
  </widget>
  <widget class="QTimeEdit" name="startTimeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>118</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="classDateEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>课程日期</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
